src/taco-remote/lib/tacoRemoteConfig.ts

```diff
@@ -60,7 +63,6 @@ class TacoRemoteConfig implements RemoteBuild.IReadOnlyDictionary {
 
     /**
      * @name allowsEmulate
-     * @LOCTAG TacoRemoteConfigAllowsEmulate
```

Comment:
Yes, when I was adding the new property I noticed that these were unused and so I cleaned them up.